dataeng/resources/vault-config.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+
+# make a directory to store vault token helper and config file
+mkdir ${WORKSPACE}/vault-config
+# get token helper from repo
+cp ${WORKSPACE}/jenkins-job-dsl/dataeng/resources/token-helper.py ${WORKSPACE}/vault-config/token-helper.py
```

Comment:
Instead of copying it, you can just use the path to the file in the repo and save yourself a step